zone/zonedb.cpp

```diff
@@ -4076,3 +4076,25 @@ int16 ZoneDatabase::GetTimerFromSkill(EQ::skills::SkillType skillid)
 
 	return timer;
 }
+
+bool ZoneDatabase::GetZoneBanishPoint(ZoneBanishPoint* into_zbp, const char* dest_zone) {
+	std::string query = StringFormat("SELECT zone, pos_x, pos_y, pos_z, heading, dest_zone "

```

Comment:
This query needs to reference 5 new fields you add to the zone table, and should never reference the book directly.

There are future use cases where we would want this to go to specific locations.

Reply:
Please also add the SQL to the appropriate folder when you change the above code path.